app/javascript/mastodon/features/account_timeline/containers/header_container.jsx

```diff
@@ -54,15 +53,6 @@ const mapDispatchToProps = (dispatch, { intl }) => ({
           onConfirm: () => dispatch(unfollowAccount(account.get('id'))),
         },
       }));
-    } else if (account.getIn(['relationship', 'requested'])) {
-      dispatch(openModal({
-        modalType: 'CONFIRM',
-        modalProps: {
-          message: <FormattedMessage id='confirmations.cancel_follow_request.message' defaultMessage='Are you sure you want to withdraw your request to follow {name}?' values={{ name: <strong>@{account.get('acct')}</strong> }} />,
-          confirm: intl.formatMessage(messages.cancelFollowRequestConfirm),
-          onConfirm: () => dispatch(unfollowAccount(account.get('id'))),
-        },
-      }));
```

Comment:
Why remove this case?

Reply:
I think that "withdraw request"/"cancel follow" is a bit confusing and "unfollow" makes more sense to be universal.

Reply:
I'm not sure it is more confusing, and I think it makes more sense to spell out exactly what you're doing. But I won't die on this hill. The rest of the PR looks fine to me.